SparkPost/2web2ui
# FE-1134: Update babel to include optional-chaining

### What Changed
 - Updated babel core and babel preset

### How To Test
 - Verify that the codebase compiles with npm start
 - Verify that there are no more instances of `({...foo}) => {...}`
